### ISSUE TITLE
Specify Project explorer in installation steps

### ISSUE THREAD

**Issue by jonadv** [hilkoc/vbaDeveloper#55]
Please adjust this line in Build.bas 

```
' 5. If using a non-English version of Excel, rename your current workbook into ThisWorkbook (in VB Editor, press F4,
'    then under the local name for Microsoft Excel Objects, select the workbook. Set the property '(Name)' to ThisWorkbook)

```
to

```
' 5. If using a non-English version of Excel, rename your current workbook into ThisWorkbook (in VB Editor, **press Ctrl+R**,
'    then **in the Project Explorer** under the local name for Microsoft Excel Objects, select the workbook. Set the property '(Name)' to ThisWorkbook)
```

 - F4 opens the Properties Window, not the Project explorer! 
 - 'Microsoft Excel Objects' sounds like a very general thing which I have always overlooked the passed 5 years. Took a while before I realised it is just in the Project Explorer, and not in the Properties Window as instructed..

![image](https://user-images.githubusercontent.com/10421216/121560554-ee89d100-ca17-11eb-8936-68814b62b088.png)

**Comment by hilkoc** [comment]
Thanks for your comment. Happy to accept a pull request for this.

**Comment by jonadv** [comment]
K I will make it soon

**Comment by jonadv** [comment]
would you agree that step 3 would improve if it is changed from 

```

' 3. Rename the project to 'vbaDeveloper' 
```

to:
```

' 3. Rename the project to 'vbaDeveloper' (in Project Explorer (Ctrl+R to show) select current project and click F4 
'    to open Properties window. Then set property '(Name)' to 'vbaDeveloper')
```

**Comment by jonadv** [comment]
I'm adjusting a bit more, hope you agree. I got stuck on step 7 as well. Tried to save in the directory 'src', but there is a folder named vbaDeveloper.xlam, which doesnt let you save a file with the same name there. Only later I noticed it sais in the SAME directory as 'src', not in the directory 'src'. Just to amplify that I add "and the 'README.md' file": 

' 7. In VB Editor, menu File-->Save Book1; Save as vbaDeveloper.xlam in the same directory as 'src' and the 'README.md' file
